### PR TITLE
Fix 151

### LINKED ISSUE #151: Some functions need deprecating
There are a bunch of functions in the `models.py` script that have a deprecation warning decorator. We should get rid of these.

@joaomoura24, if you have code that is using these functions it will break when we remove these functions. For each function being deprecated there is an alternative (with the correct naming convention). I will remove these and setup a PR with you as reviewer.

### PR DESCRIPTION
**Pull request by cmower**
# Description

Resolves #151.

# Todo

- [x] Removed deprecated functions.
- [x] Ensure all unit tests pass.
- [x] Address all issues raised by unit-tests.
- [x] Added code reviewer.

### CODE REVIEW

**Comment by cmower** [comment]
@joaomoura24 please let me know when we can merge this PR.

File: optas/models.py [hunk 1]
```diff
@@ -54,29 +52,6 @@ def listify(*args, **kwargs):
...
-
-            args_use = list(args)
-            if function.__name__.endswith("_function"):
-                args_use = args_use[1:]
-
-            return new_function(*args_use, **kwargs)
-
-        return wrapper
-
-    return decorator
```
Comment (joaomoura24) [inline optas/models.py:77]: Wouldn't be better to leave this decorator for future use?

Comment (cmower) [reply optas/models.py]: Good point, reverted in cd44b3d19506181f09377f2460e6d4da7eedd789.

**Comment by joaomoura24** [comment]
For me you can merge once the checks are done.

**Pull request merged**